src/components/BlogRoll.js

```diff
@@ -12,19 +13,37 @@ class BlogRoll extends React.Component {
         {posts &&
           posts.map(({ node: post }) => (
             <div className="is-parent column is-6" key={post.id}>
-              <article className="tile is-child box notification">
-                <p>
-                  <Link
-                    className="title has-text-primary is-size-4"
-                    to={post.fields.slug}
-                  >
-                    {post.frontmatter.title}
-                  </Link>
-                  <span> &bull; </span>
-                  <span className="subtitle is-size-5 is-block">
-                    {post.frontmatter.date}
-                  </span>
-                </p>
+              <article
+                className={`blog-list-item tile is-child box notification ${
+                  post.frontmatter.featured ? 'is-featured' : ''
```

Comment:
Can you clarify what's the role of the `post.frontmatter.featured`, i'm not seeing it in the markdown only `featuredpost` property. Since `featured` is allways null the`'is-featured'` css class is not getting added to the markdown, and even if i add it manually using Google Chrome it doesn't have any effect

Reply:
Yep, this is your classic dev typo - d'oh! Let me update the code (you're correct, it should be 'featuredpost') and it'll update the PR.

Reply:
No, worries, this is awesome!